x-pack/plugins/security_solution/public/management/pages/policy/models/policy_details_config.ts

```diff
@@ -69,7 +32,7 @@ export const setIn = (a: UIPolicyConfig) => <Key extends keyof UIPolicyConfig>(k
 >(
   v: V
```

Comment:
This function's logic is short, so no need to change. but normally, I would suggest using more descriptive variables names instead of a single letter like `v` or `c` below :)